packages/discord.js/typings/index.d.ts

```diff
@@ -4211,6 +4211,11 @@ export class GuildApplicationCommandManager extends ApplicationCommandManager<Ap
   public set(commands: readonly ApplicationCommandDataResolvable[]): Promise<Collection<Snowflake, ApplicationCommand>>;
 }
 
+export interface FollowedChannel {
+  channelId: Snowflake;
+  webhookId: Snowflake;
+}
```

Comment:
IMO we should pick a different name for this, since `FollowedChannel` follows the pattern we use for actual channels, i.e., `TextChannel`, which might mislead users

Something like `FollowedChannelData` could perhaps work